docs/docs/08.1-more-about-refs.md

```diff
@@ -1,13 +1,97 @@
 ---
 id: more-about-refs
-title: More About Refs
+title: References to Components
 permalink: more-about-refs.html
 prev: working-with-the-browser.html
 next: tooling-integration.html
 ---
-After returning the structure of your UI from the render method, you may find yourself wanting to "reach out" and invoke methods on component instances returned from `render()`. Often, doing something like this isn't necessary for making data flow through your application, because the Reactive data flow always ensures that the most recent `props` are sent to each child that is output from `render()`. However, there are a few cases where it still might be necessary or beneficial.
+After building your component, you may find yourself wanting to "reach out" and invoke methods on component instances returned from `render()`.  In most cases, this is ideally unnecessary (and should be avoided) because the Reactive data flow always ensures that the most recent props are sent to each child that is output from render().  However, there are a few cases where it still might be necessary or beneficial, so React provides an escape hatch known as `refs`.  These `refs` (references) are especially useful when you need to find the DOM markup rendered by a component (for instance, to position it absolutely), when using React components in a larger non-React application, or when transitioning your code to React.
 
-Consider the case, when you wish to tell an `<input />` element (that exists within your instances sub-hierarchy) to focus after you update its value to be the empty string, `''`.
+Consider the case, when you wish to tell an `<input />` element (that exists within your instances sub-hierarchy) to focus after you update its value to be the empty string, `''`.  That is to say, we want to implement an imperative function that will reset your application's input field to the empty state.  This is a reasonable use case for refs, and one which we will explore in a moment, but first we'll look at the various options for getting a reference.
+
+## The ref returned from React.render
```

Comment:
We should move http://facebook.github.io/react/docs/working-with-the-browser.html#refs-and-finddomnode into this page instead.